Please run `msftidy.rb` and address issues.

```
[*] Running msftidy.rb in ./.git/hooks/post-merge mode
--- Checking new and changed module syntax with tools/dev/msftidy.rb ---
modules/exploits/windows/local/bypassuac_sluihijack.rb - [ERROR] Incorrect disclosure date format
modules/exploits/windows/local/bypassuac_sluihijack.rb:37 - [WARNING] Spaces at EOL
```

--- a/modules/exploits/windows/local/bypassuac_sluihijack.rb
+++ b/modules/exploits/windows/local/bypassuac_sluihijack.rb
@@ -118,7 +118,7 @@ class MetasploitModule < Msf::Exploit::Local
       return
     end
 
-    payload_value = rand_text_alpha(12)
+    payload_value = rand_text_alpha(8)
     psh_path = expand_path(psh_path)
 
     template_path = Rex::Powershell::Templates::TEMPLATE_DIR

--- a/modules/exploits/windows/local/bypassuac_sluihijack.rb
+++ b/modules/exploits/windows/local/bypassuac_sluihijack.rb
@@ -28,14 +28,17 @@ class MetasploitModule < Msf::Exploit::Local
         info,
         'Name'          => 'Windows UAC Protection Bypass (Via Slui File Handler Hijack)',
         'Description'   => %q{
-          slui.exe is an auto-elevated binary that is vulnerable to file handler hijacking.
+          This module will bypass UAC on Windows 8-10 by hijacking a special key in the Registry under
+          the Current User hive, and inserting a custom command that will get invoked when any binary (.exe)
+          application is launched. But slui.exe is an auto-elevated binary that is vulnerable to file handler hijacking.
+          When we run slui.exe with changed Registry key (HKCU:\Software\Classes\exefile\shell\open\command),
+          it will run our custom command as Admin instead of slui.exe.
 
-          Read access to HKCU\Software\Classes\exefile\shell\open is performed upon execution.
-          Due to the registry key being accessible from user mode, an arbitrary executable file can be injected.
+          The module modifies the registry in order for this exploit to work. The modification is reverted
+          once the exploitation attempt has finished.
 
-          This exploit is generally independent from programming language and bitness, as no DLL injection or
-          privileged file copy is needed. In addition, if default system binaries suffice, file drops can be
-          avoided altogether.
+          The module does not require the architecture of the payload to match the OS. If specifying EXE::Custom
+          your DLL should call ExitProcess() after starting the payload in a different process.
         },
         'License'       => MSF_LICENSE,
         'Author'        => [

--- a/modules/exploits/windows/local/bypassuac_sluihijack.rb
+++ b/modules/exploits/windows/local/bypassuac_sluihijack.rb
@@ -34,7 +34,7 @@ class MetasploitModule < Msf::Exploit::Local
           Due to the registry key being accessible from user mode, an arbitrary executable file can be injected.
 
           This exploit is generally independent from programming language and bitness, as no DLL injection or
-          privileged file copy is needed. In addition, if default system binaries suffice, file drops can be 
+          privileged file copy is needed. In addition, if default system binaries suffice, file drops can be
           avoided altogether.
         },
         'License'       => MSF_LICENSE,
@@ -55,7 +55,7 @@ class MetasploitModule < Msf::Exploit::Local
             'URL', 'https://github.com/gushmazuko/WinBypass/blob/master/SluiHijackBypass.ps1'
           ]
         ],
-        'DisclosureDate' => 'January 15 2018'
+        'DisclosureDate' => 'Jan 15 2018'
       )
     )
   end
@@ -118,7 +118,7 @@ class MetasploitModule < Msf::Exploit::Local
       return
     end
 
-    payload_value = rand_text_alpha(8)
+    payload_value = rand_text_alpha(12)
     psh_path = expand_path(psh_path)
 
     template_path = Rex::Powershell::Templates::TEMPLATE_DIR

